spec/features/interactions/standard_interaction_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+require './db/sample_data/users'
+require './db/sample_data'
+
+RSpec::Steps.steps "Demo" do
+  it 'loads the seeds and populate with sample data' do
+    Rails.application.load_seed
+    add_sample_data
+  end
+
+  it "should show the welcome page on startup" do
+    visit root_path
+    expect(page).to have_text I18n.t('start_page.welcome_to')
+  end
+
+  it 'should let an organizer log in' do
+    #TODO: Use id instead of visible string, currently id is missing for login button
+    click_link I18n.t('navbar.login')
+    login_with_credentials(user_organizer.email, user_password)
+    expect(page).to_not have_text I18n.t('devise.failure.inactive')
+    expect(page).to_not have_text I18n.t('devise.failure.invalid')
+    expect(page).to_not have_text I18n.t('devise.failure.not_found_in_database')
+    expect(page).to_not have_text I18n.t('devise.failure.locked')
+    expect(page).to have_text I18n.t('start_page.welcome_to')
```

Comment:
Wouldn't it be better to check for the `sessions.signed_in` message instead?
If you do want to leave it that way, I would include `shows the start page` in the test case description.